src/components/avatars/index.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div>
+      <!-- default -->
+      <div
+      v-if="type == 'default'">
+      <img
+        v-bind="$attrs"
+            :class="{
+            'dd-h-6 dd-w-6': size === 'mini',
+            'dd-h-8 dd-w-8': size === 'small',
+            'dd-h-10 dd-w-10': size === 'medium',
+            'dd-h-12 dd-w-12': size === 'large',
+            'dd-h-14 dd-w-14': size === 'xLarge',
+            }"
+            class="dd-inline-flex dd-inline-block dd-rounded-full" 
+            :src="srcLink">
```

Comment:
In case if user doesn't give you image src set by default letter name

Reply:
And this issue is also resolved now.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/85391382/217249520-ea702ecd-6449-49db-9e1b-f3f079221f68.png">